docs/tutorials/scala/mxnet_scala_on_intellij.md

```diff
@@ -347,6 +379,30 @@ Library not loaded: /usr/local/opt/opencv/lib/libopencv_calib3d.x.x.dylib
 This can be resolved be installing OpenCV.
 
 
+### Using MXNet from source
+
+If you chose to "Build from Source" when following the [install instructions](https://mxnet.incubator.apache.org/install/index.html), you can use your custom build instead of the build from maven.  Use your build by editing the `pom.xml` file and replacing the `org.apache.mxnet` dependency with the following:
```

Comment:
There's this resource too: https://github.com/apache/incubator-mxnet/blob/master/docs/install/build_from_source.md

Reply:
And this table for bindings by OS. https://github.com/apache/incubator-mxnet/blob/master/docs/install/build_from_source.md#installing-mxnet-language-bindings